src/store/servers.ts

```diff
@@ -36,27 +36,11 @@ export const servers: IServer[] = [
       PersistShines : { Enabled: true },
     },
   },
-  {
-    name     : 'Piplup (UDP)',
-    link     : 'https://piplup.smoo.it',
-    server   : { host: 'piplus.smoo.it', ip: '34.29.243.224', port: 1029 },
-    location : { flag: 'us', name: 'US-Central' },
-    settings : {
-      Server        : { MaxPlayers: 8 },
-      Scenario      : { MergeEnabled: true },
-      Shines        : { Enabled: true },
-      PersistShines : { Enabled: true },
-    },
-    version: {
-      html    : linkRelease('1.0.5-rcl.1', 'udp/1.0.5', 'Istador/smo-multi-rs'),
-      details : `<p>Connecting requires mod version ${linkRelease('v1.4.0', 'v1.4.0', 'CraftyBoss/SuperMarioOdysseyOnline')} or later.</p>`,
-    },
-  },
   {
     name     : 'Muz',
     server   : { host: 'larbe.org', ip: '49.13.203.68' },
     location : { flag: 'de', name: 'Germany' },
-    version  : linkTree('1.0.5-rcl.8', 'Muz-dev/SmoOnlineServer'),
+    version  : linkTree('1.0.5-rcl.8', 'Muz-dev/json-api', 'Muz-dev/SmoOnlineServer'),
```

Comment:
This should either link to the release which has artifacts to download for others interested or name the entry correcly as a branch instead of a release.

```suggestion
    version  : linkRelease('1.0.5-rcl.8', 'rcl/1.0.5', 'Istador/SmoOnlineServer'),
```
or

```suggestion
    version  : linkTree('Muz/json-api', 'Muz-dev/json-api', 'Muz-dev/SmoOnlineServer'),
```

The [only difference](https://github.com/Istador/SmoOnlineServer/compare/json-api...Muz-dev:SmoOnlineServer:json-api) is on the `README.md`, so it's not really another version afaik, so I [changed](https://github.com/Istador/smoo.it/commit/2aa26fd0094983a54d69ef7cfce5abf0153291d3) it already to link to my [release](https://github.com/Istador/SmoOnlineServer/releases/tag/1.0.5-rcl.8).